cmd/lotus-miner/sealing.go

```diff
@@ -101,17 +101,27 @@ var sealingWorkersCmd = &cli.Command{
 
 			ramBarsRes := int(stat.Info.Resources.MemReserved * barCols / stat.Info.Resources.MemPhysical)
 			ramBarsUsed := int(stat.MemUsedMin * barCols / stat.Info.Resources.MemPhysical)
-			ramBar := color.YellowString(strings.Repeat("|", ramBarsRes)) +
-				color.GreenString(strings.Repeat("|", ramBarsUsed)) +
-				strings.Repeat(" ", int(barCols)-ramBarsUsed-ramBarsRes)
+			ramRepeatSpace := int(barCols) - (ramBarsUsed + ramBarsRes)
+			var ramBar string
+			if ramRepeatSpace < 0 {
+				ramRepeatSpace = 0
+				ramBar = color.RedString(strings.Repeat("|", ramBarsRes)) +
+					color.GreenString(strings.Repeat("|", ramBarsUsed)) +
+					strings.Repeat(" ", ramRepeatSpace)
+			} else {
+				ramBar = color.YellowString(strings.Repeat("|", ramBarsRes)) +
+					color.GreenString(strings.Repeat("|", ramBarsUsed)) +
+					strings.Repeat(" ", ramRepeatSpace)
+			}
```

Comment:
nit: populate the variable portion of `ramBar` under the condition, then the static version outside.

Reply:
done